src/django/api/mail.py

```diff
@@ -48,20 +48,19 @@ def send_claim_facility_confirmation_email(request, facility_claim):
         'facility_country': facility_country,
         'facility_url': make_facility_url(request, facility_claim.facility),
         'contact_person': facility_claim.contact_person,
-        'email': facility_claim.email,
+        'email': facility_claim.contributor.admin.email,
```

Comment:
A bit unsafe construction. I would add optional chainning here because property has a few nesting levels.

Reply:
I don't think we need to do that because of the database constraints. It's impossible to have a facility claim without a contributor, a contributor without an admin, or an admin without an email.

Reply:
@VadimKovalenkoSNF